jcache/commons/src/test/java/org.infinispan.jcache/AbstractTwoCachesAnnotationsTest.java

```diff
@@ -0,0 +1,108 @@
+package org.infinispan.jcache;
```

Comment:
The folder where this file is located might be wrong? It shows as being `jcache/commons/src/test/java/org.infinispan.jcache` when it should be `jcache/commons/src/test/java/org/infinispan/jcache`